src/modules/navigator/rtl_direct.cpp

```diff
@@ -368,7 +368,9 @@ void RtlDirect::set_rtl_item()
 
 			_mission_item.land_precision = _param_rtl_pld_md.get();
 
-			startPrecLand(_mission_item.land_precision);
+			if (_mission_item.land_precision > 0) {
```

Comment:
How does it work with the RTL. Is the precision land request still in the mission item or is it in the RTL_PLD_MD parameter (or do you need to consider both? and or or?)

Reply:
Even thinking about it more, the _mission_item.land_precision is not set in the RTL direct